lib/Backends/OpenCL/Transforms.cpp

```diff
@@ -58,7 +37,7 @@ bool OCLBackend::transformPostLowering(Function *F, CompilationMode mode) {
     if (auto *CN = dyn_cast<ConvolutionNode>(&node)) {
       // Note: Using OCL-specific convolution conversion here since OCL only
```

Comment:
This comment is stale and can be removed now that `convertConvToNCHWConv()` is used